crypto/BRCryptoTransfer.c

```diff
@@ -393,6 +393,33 @@ cryptoTransferGetAmountDirected (BRCryptoTransfer transfer) {
     return amount;
 }
 
+extern BRCryptoAmount
+cryptoTransferGetAmountDirectedNet (BRCryptoTransfer transfer) {
+    BRCryptoAmount amount = cryptoTransferGetAmountDirected (transfer);
+
+    // If the transfer->unit and transfer->unitForFee differ then there is no fee
+    if (cryptoUnitIsIdentical (transfer->unit, transfer->unitForFee))
+        return amount;
+
+    BRCryptoFeeBasis feeBasis = (NULL != transfer->feeBasisConfirmed
```

Comment:
TSAN issue here, likely.

Reply:
This would be the same TSAN issue suffered by: `cryptoTransferGetAmountDirected()`, `cryptoTransferGetAmount()`, etc?  That is, the issue is not specific to `cryptoTransferGetAmountDirectedNet()`?

Reply:
Is so, I'll add a TSAN issue.

Reply:
I updated CORE-462